controllers/ocirepository_controller.go

```diff
@@ -147,6 +148,7 @@ func (r *OCIRepositoryReconciler) SetupWithManagerAndOptions(mgr ctrl.Manager, o
 		WithOptions(controller.Options{
 			MaxConcurrentReconciles: opts.MaxConcurrentReconciles,
 			RateLimiter:             opts.RateLimiter,
+			RecoverPanic:            recoverPanic,
```

Comment:
@Santosh1176 I had a quick chat with @darkowlzz on this, and we agreed that we can go ahead and use the literals instead. Given that, this is only used one per controllers and we have no intention to toggle or change this value. So each controller would have this instead:

```suggestion
			RecoverPanic:            true,
```